src/System.ServiceModel.Security/ref/System.ServiceModel.Security.csproj

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
+    <AssemblyVersion>4.0.2.0</AssemblyVersion>
```

Comment:
You could just change https://github.com/StephenBonikowsky/wcf/blob/45229d8763582d6e8ff8edb1bd1fc12db33af575/src/System.ServiceModel.Security/dir.props#L5